client/src/index.css

```diff
@@ -26,7 +26,7 @@ code {
   }
 
   .filled-btn {
-    @apply text-white bg-primary hover:text-primaryDark hover:bg-primaryLight font-medium rounded-sm text-sm transition-all;
+    @apply text-white px-5 py-2.5 text-center bg-primary hover:text-primaryDark hover:bg-primaryLight font-medium rounded-md text-sm transition-all;
```

Comment:
you can overwrite style in className and shouldn't constantly change the tailwind defined class, cause it will affect to many component and lead to unexpected UI change